test/ct_test.c

```diff
@@ -124,6 +124,7 @@ static CT_TEST_FIXTURE set_up(const char *const test_case_name)
 static void tear_down(CT_TEST_FIXTURE fixture)
 {
     CTLOG_STORE_free(fixture.ctlog_store);
+    SCT_free(fixture.sct);
```

Comment:
I was wondering about that.. FYI, ct_test was written based on https://wiki.openssl.org/index.php/How_To_Write_Unit_Tests_For_OpenSSL.